spyder/plugins/explorer/widgets/tests/test_explorer.py

```diff
@@ -171,5 +171,8 @@ def run_test_helper(single_click, initial_index):
     run_test_helper(single_click=False, initial_index=initial_index)
 
 
+def test_assocaitions(qtbot, file_explorer):
+
```

Comment:
Please add something here or at least a `pass` to make our tests run correctly (this is giving an indentation error right now).